examples/amp-story/player-amp.html

```diff
@@ -17,19 +17,13 @@ <h1>This is an AMP page that embeds a story below:</h1>
             class="story">
           <span class="title">A local’s guide to what to eat and do in New York City</span>
         </a>
-        <a href="https://www-washingtonpost-com.cdn.ampproject.org/v/s/www.washingtonpost.com/graphics/2019/lifestyle/travel/amp-stories/a-locals-guide-to-what-to-eat-and-do-in-miami/"
-            data-poster-portrait-src="https://www-washingtonpost-com.cdn.ampproject.org/v/s/www.washingtonpost.com/graphics/2019/lifestyle/travel/amp-stories/a-locals-guide-to-what-to-eat-and-do-in-miami/img/promo3x4.jpg"
-            class="story">
+        <a href="https://www-washingtonpost-com.cdn.ampproject.org/v/s/www.washingtonpost.com/graphics/2019/lifestyle/travel/amp-stories/a-locals-guide-to-what-to-eat-and-do-in-miami/" class="story">
```

Comment:
Can we add `img loading=lazy` here too, and update the player validation rules?

Reply:
Done.

Reply:
👀  please add an `img loading=lazy`